samples/v1.0/Tests/FactSet.Test.json

```diff
@@ -0,0 +1,74 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.0",
+	"body": [
+		{
+			"type": "FactSet",
+			"facts": [
+				{
+					"title": "This is a long FactSet that should wrap the title",
+					"value": "This is a long FactSet that can introduce wrapping for the value"
+				},
+				{
+					"title": "Fact 2",
+					"value": "Value 2"
+				},
+				{
+					"title": "Fact 3",
+					"value": "Value 3"
+				},
+				{
+					"title": "Fact 4",
+					"value": "Value 5"
+				}
+			]
+		},
+		{
+			"type": "TextBlock",
+			"text": "FactSet with missing facts property is below",
+			"spacing": "None",
+			"wrap": true
+		},
+		{
+			"type": "FactSet"
+		},
+		{
+			"type": "TextBlock",
+			"text": "FactSet with 0 facts is below",
+			"spacing": "None",
+			"wrap": true
+		},
+		{
+			"type": "FactSet",
+			"facts": []
+		},
+		{
+			"type": "TextBlock",
+			"text": "FactSet with facts wihout Title, without Value, wihout both, and a random property is below",
```

Comment:
>wihout [](start = 31, length = 6)

typo (without)

Reply:
thanks

---
In reply to: [494598392](https://github.com/microsoft/AdaptiveCards/pull/4809#discussion_r494598392) [](ancestors = 494598392)